code/processes/wdb.q

```diff
@@ -262,17 +266,17 @@ endofdaysortdate:{[dir;pt;tablist;hdbsettings]
 	/-sort permitted tables in database
 	/- sort the table and garbage collect (if enabled)
 	.lg.o[`sort;"starting to sort data"];
-	$[count[.z.pd[]]&0>system"s";
+	$[(0 < count .z.pd[]) and ((system "s")<0);
```

Comment:
Can use the original without comparison: `count[.z.pd[]]&system"s"` or revert to the way it was.